packages/server/app/Controllers/Ws/MpeRoomsWsController.ts

```diff
@@ -374,4 +382,24 @@ export default class MpeRoomsWsController {
             throw new Error('onGetContext error');
         }
     }
+
+    public static async onExportToMtv({
+        roomID,
+        userID,
+        deviceID,
+        mtvRoomOptions,
+    }: OnExportToMtvArgs): Promise<void> {
+        try {
+            await MpeServerToTemporalController.exportToMtv({
```

Comment:
Unless I'm mistaken only mpe room members should be able to export the mpe as mtv ? if true involves one more test

Reply:
Implemented in last commits 👍